WebPhapp/backend/mysql_scripts/pharmacopeia.sql

```diff
@@ -0,0 +1,27 @@
+/*
+ * Create the Pharmacopeia table and insert some items. ID starts at 11.
+ * Ran on AWS on 2/1/19
+ */
+
+USE seniordesign1;
```

Comment:
Why is this named seniordesign1?

Reply:
So that we can create multiple databases and keep this one as a historical. Mainly because I might use my AWS instance for other things than just senior design so I wanted the name to be obvious.